.codeclimate.yml

```diff
@@ -15,3 +15,4 @@ exclude_paths:
   - internal/gps/_testdata
   - cmd/dep/testdata
   - testdata
+  - internal/gps/internal/pb
```

Comment:
Something like `**.pb.go` (if legal here) would be more general and continue to work as things change.